diff --git a/assets/js/alert.js b/assets/js/alert.js
index 17f1dd2..568522b 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -8,7 +8,7 @@ export default class {
   render () {
     const element = document.createElement('div');
     element.classList.add('alert', 'alert-success', 'mt-alert');
-    element.innerHTML = `<span class="fa fa-lightbulb"></span> ${message}`;
+    element.innerHTML = `<span class="fa fa-lightbulb"></span> ${this.message}`;
     document.body.insertAdjacentElement('afterbegin', element);
   }
 }
