diff --git a/assets/js/alert.js b/assets/js/alert.js
index 568522b..7f1d068 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -6,9 +6,16 @@ export default class {
   }
 
   render () {
-    const element = document.createElement('div');
-    element.classList.add('alert', 'alert-success', 'mt-alert');
-    element.innerHTML = `<span class="fa fa-lightbulb"></span> ${this.message}`;
-    document.body.insertAdjacentElement('afterbegin', element);
+    this.element = document.createElement('div');
+    this.element.classList.add('alert', 'alert-success', 'mt-alert');
+    this.element.innerHTML = `<span class="fa fa-lightbulb"></span> ${this.message}`;
+    document.body.insertAdjacentElement('afterbegin', this.element);
+  }
+
+  /**
+   * @returns {boolean}
+   */
+  isVisible () {
+    return Boolean(this.element);
   }
 }
diff --git a/assets/js/app.js b/assets/js/app.js
index aa542c2..92640d3 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -21,4 +21,5 @@ $('.js-comment-vote').on('click', async function (e) {
   $container.find('.js-vote-total').text(data.votes);
   const voteAlert = new Alert('Vote counted!');
   voteAlert.render();
+  console.log(voteAlert.isVisible());
 });
