diff --git a/assets/js/app.js b/assets/js/app.js
index 4c77c69..d59b325 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -1,4 +1,4 @@
-import notify from './notify';
+import Notification from './notify';
 import $ from 'jquery';
 import '../css/app.css';
 
@@ -19,5 +19,6 @@ $('.js-comment-vote').on('click', async function (e) {
   });
 
   $container.find('.js-vote-total').text(data.votes);
-  notify('Vote counted!');
+  const notification = new Notification('Vote counted!');
+  notification.render();
 });
