diff --git a/web/assets/js/app.js b/web/assets/js/app.js
index 8cd16d1..ec29728 100644
--- a/web/assets/js/app.js
+++ b/web/assets/js/app.js
@@ -6,15 +6,14 @@ $('.js-comment-vote').on('click', function (e) {
   const $button = $(e.currentTarget);
   const $container = $button.closest('.js-vote-arrows');
 
-  $.ajax({
+  const data = await $.ajax({
     url: $container.data('url'),
     data: JSON.stringify({
       direction: $button.data('direction')
     }),
     method: 'POST'
-  }).then(function (data) {
-    $container.find('.js-vote-total').text(data.votes);
-
-    alert('Vote counted!');
   });
+
+  $container.find('.js-vote-total').text(data.votes);
+  alert('Vote counted!');
 });
