diff --git a/city.js b/city.js
index 84e8ca4..0e0e3ec 100644
--- a/city.js
+++ b/city.js
@@ -1,3 +1,9 @@
+/**
+ * @typedef {Object} Stats
+ * @property {string} name
+ * @property {number} population
+ */
+
 module.exports = class {
   constructor (name) {
     this._name = name;
@@ -25,7 +31,7 @@ module.exports = class {
   }
 
   /**
-   * @returns {Promise<Object>}
+   * @returns {Promise<Stats>}
    */
   async printStats () {
     const stats = await this.getStats();
