diff --git a/assets/js/notify.js b/assets/js/notify.js
index 0c7c863..eb65077 100644
--- a/assets/js/notify.js
+++ b/assets/js/notify.js
@@ -1,6 +1,6 @@
 export default function (message) {
   const element = document.createElement('div');
   element.classList.add('alert', 'alert-success');
-  element.innerHTML = '<span class="fa fa-lightbulb"></span> ' + message;
+  element.innerHTML = `<span class="fa fa-lightbulb"></span> ${message}`;
   document.body.insertAdjacentElement('afterbegin', element);
 }
