diff --git a/assets/css/alert.css b/assets/css/alert.css
new file mode 100644
index 0000000..661c5c7
--- /dev/null
+++ b/assets/css/alert.css
@@ -0,0 +1,3 @@
+.alert.mt-alert {
+    margin: 0;
+}
diff --git a/assets/js/alert.js b/assets/js/alert.js
index eb65077..4a44a4b 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -1,6 +1,8 @@
+import '../css/alert.css';
+
 export default function (message) {
   const element = document.createElement('div');
-  element.classList.add('alert', 'alert-success');
+  element.classList.add('alert', 'alert-success', 'mt-alert');
   element.innerHTML = `<span class="fa fa-lightbulb"></span> ${message}`;
   document.body.insertAdjacentElement('afterbegin', element);
 }
