diff --git a/web/assets/css/app.css b/web/assets/css/app.css
index 85bc3f6..8b3666b 100644
--- a/web/assets/css/app.css
+++ b/web/assets/css/app.css
@@ -1,3 +1,4 @@
+@import '~bootstrap/dist/css/bootstrap.css';
 body {
     font-family: spartan;
     color: #444;
diff --git a/web/assets/js/app.js b/web/assets/js/app.js
index 155b987..8995437 100644
--- a/web/assets/js/app.js
+++ b/web/assets/js/app.js
@@ -1,6 +1,5 @@
 import sendAlert from './alert';
 import $ from 'jquery';
-import 'bootstrap/dist/css/bootstrap.css';
 import '../css/app.css';
 
 /**
