diff --git a/assets/js/alert.js b/assets/js/alert.js
index 9ca5269..e63e224 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -32,7 +32,7 @@ export default class Alert {
     document.body.insertAdjacentElement('afterbegin', this.element);
 
     this.element.querySelectorAll('[data-dismiss]').forEach((el) => {
-      el.addEventListener('click', this.handleClickClose);
+      el.addEventListener('click', this.handleClickClose.bind(this));
     });
   }
 
