diff --git a/assets/js/alert.js b/assets/js/alert.js
index 10a3ab3..b2f4221 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -20,10 +20,14 @@ export default class Alert {
     this.element.classList.add(
       'alert',
       `alert-${this.options.mode === 'success' ? 'success' : 'danger'}`,
+      'alert-dismissible',
       'mt-alert'
     );
     this.element.innerHTML = `
       <span class="fa fa-lightbulb"></span> ${this.message}
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
     `;
     document.body.insertAdjacentElement('afterbegin', this.element);
   }
