diff --git a/assets/js/alert.js b/assets/js/alert.js
index 469aac9..9ca5269 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -32,9 +32,7 @@ export default class Alert {
     document.body.insertAdjacentElement('afterbegin', this.element);
 
     this.element.querySelectorAll('[data-dismiss]').forEach((el) => {
-      el.addEventListener('click', (event) => {
-        this.handleClickClose(event);
-      });
+      el.addEventListener('click', this.handleClickClose);
     });
   }
 
