diff --git a/assets/js/alert.js b/assets/js/alert.js
index 6207f95..10a3ab3 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -22,7 +22,9 @@ export default class Alert {
       `alert-${this.options.mode === 'success' ? 'success' : 'danger'}`,
       'mt-alert'
     );
-    this.element.innerHTML = `<span class="fa fa-lightbulb"></span> ${this.message}`;
+    this.element.innerHTML = `
+      <span class="fa fa-lightbulb"></span> ${this.message}
+    `;
     document.body.insertAdjacentElement('afterbegin', this.element);
   }
 
