diff --git a/assets/js/alert.js b/assets/js/alert.js
index 7c5be5f..469aac9 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -33,13 +33,17 @@ export default class Alert {
 
     this.element.querySelectorAll('[data-dismiss]').forEach((el) => {
       el.addEventListener('click', (event) => {
-        event.preventDefault();
-
-        this.element.remove();
+        this.handleClickClose(event);
       });
     });
   }
 
+  handleClickClose (event) {
+    event.preventDefault();
+
+    this.element.remove();
+  }
+
   /**
    * @returns {boolean}
    */
