diff --git a/assets/js/alert.js b/assets/js/alert.js
index 640310a..ef998ce 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -31,8 +31,8 @@ export default class Alert {
     `;
     document.body.insertAdjacentElement('afterbegin', this.element);
 
-    this.element.querySelectorAll('[data-dismiss]').forEach(function (el) {
-      el.addEventListener('click', function () {
+    this.element.querySelectorAll('[data-dismiss]').forEach((el) => {
+      el.addEventListener('click', () => {
         this.element.remove();
       });
     });
