diff --git a/assets/js/alert.js b/assets/js/alert.js
index ef998ce..7c5be5f 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -32,7 +32,9 @@ export default class Alert {
     document.body.insertAdjacentElement('afterbegin', this.element);
 
     this.element.querySelectorAll('[data-dismiss]').forEach((el) => {
-      el.addEventListener('click', () => {
+      el.addEventListener('click', (event) => {
+        event.preventDefault();
+
         this.element.remove();
       });
     });
