diff --git a/assets/js/alert.js b/assets/js/alert.js
index 7991c94..b044acb 100644
--- a/assets/js/alert.js
+++ b/assets/js/alert.js
@@ -3,7 +3,11 @@ import '../css/alert.css';
 export default class {
   constructor (message, options) {
     this.message = message;
-    this.options = options;
+
+    const defaultOptions = {
+      mode: 'success'
+    };
+    this.options = { ...defaultOptions, ...options };
   }
 
   render () {
diff --git a/assets/js/app.js b/assets/js/app.js
index dd9c159..4b48f43 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -19,7 +19,7 @@ $('.js-comment-vote').on('click', async function (e) {
   });
 
   $container.find('.js-vote-total').text(data.votes);
-  const voteAlert = new Alert('Vote counted!', { mode: 'success' });
+  const voteAlert = new Alert('Vote counted!');
   voteAlert.render();
   console.log(voteAlert.isVisible());
 });
