diff --git a/web/assets/js/alert.js b/web/assets/js/alert.js
index 5c59f18..d1405b3 100644
--- a/web/assets/js/alert.js
+++ b/web/assets/js/alert.js
@@ -1,3 +1,3 @@
-module.exports = function (message) {
+export default function (message) {
   alert(message);
 };
diff --git a/web/assets/js/app.js b/web/assets/js/app.js
index b6c4515..ff5548a 100644
--- a/web/assets/js/app.js
+++ b/web/assets/js/app.js
@@ -1,4 +1,4 @@
-const sendAlert = require('./alert');
+import sendAlert from './alert';
 
 /**
  * Simple (ugly) code to handle the comment vote up/down
