diff --git a/assets/js/comment.js b/assets/js/comment.js
index 991ee8a..4d83868 100644
--- a/assets/js/comment.js
+++ b/assets/js/comment.js
@@ -1,6 +1,7 @@
 import delegate from 'delegate';
 import axios from 'axios';
 import { alertSuccess, alertError } from './alert';
+import { getHappyMessage } from './random-message';
 
 export default class {
   constructor (element) {
@@ -25,6 +26,6 @@ export default class {
     }
 
     this.element.querySelector('.js-vote-total').innerHTML = response.data.votes;
-    alertSuccess('Vote counted!');
+    alertSuccess(`Vote counted! ${getHappyMessage()}`);
   }
 }
diff --git a/assets/js/random-message.js b/assets/js/random-message.js
new file mode 100644
index 0000000..63ff7f1
--- /dev/null
+++ b/assets/js/random-message.js
@@ -0,0 +1,9 @@
+export const getHappyMessage = () => {
+  const messages = [
+    'You rock!',
+    'Slow clap.',
+    'Simply amazing.'
+  ]
+
+  return messages[Math.floor(Math.random() * messages.length)];
+};
