diff --git a/city.js b/city.js
index c4979a0..1422236 100644
--- a/city.js
+++ b/city.js
@@ -1,9 +1,9 @@
 module.exports = class {
     constructor(name) {
-        this.name = name;
+        this._name = name;
     }
 
-    getName() {
-        return this.name;
+    get name() {
+        return this._name;
     }
 }
diff --git a/play.js b/play.js
index 2f02eaa..1373080 100644
--- a/play.js
+++ b/play.js
@@ -1,4 +1,4 @@
 const City = require('./city');
 
 const qro = new City('Querétaro');
-console.log(qro.getName());
+console.log(qro.name);
