diff --git a/city.js b/city.js
index 1422236..6d7c278 100644
--- a/city.js
+++ b/city.js
@@ -6,4 +6,10 @@ module.exports = class {
     get name() {
         return this._name;
     }
+
+    printNameLater() {
+        setTimeout(function() {
+            console.log(this.name);
+        }, 1000);
+    }
 }
diff --git a/play.js b/play.js
index bc0fc3e..9ecd697 100644
--- a/play.js
+++ b/play.js
@@ -2,8 +2,6 @@ const City = require('./city');
 
 const qro = new City('Querétaro');
 
-setTimeout(function() {
-    console.log(qro.name);
-}, 1000);
+qro.printNameLater();
 
 console.log('Done!');
