diff --git a/city.js b/city.js
index 30f142b..b5910ac 100644
--- a/city.js
+++ b/city.js
@@ -1 +1,9 @@
-module.exports = 'Querétaro';
+module.exports = class {
+  constructor (name) {
+    this.name = name;
+  }
+
+  getName () {
+    return this.name;
+  }
+}
diff --git a/play.js b/play.js
index 6f25871..2f02eaa 100644
--- a/play.js
+++ b/play.js
@@ -1,3 +1,4 @@
-const city = require('./city');
+const City = require('./city');
 
-console.log(city);
+const qro = new City('Querétaro');
+console.log(qro.getName());
