diff --git a/city.js b/city.js
index 1ef16a8..090bd44 100644
--- a/city.js
+++ b/city.js
@@ -19,7 +19,7 @@ module.exports = class {
 
         console.log(stats);
 
-        resolve();
+        resolve(stats);
       }, 1000);
     });
   }
diff --git a/play.js b/play.js
index 3687782..8cf5a69 100644
--- a/play.js
+++ b/play.js
@@ -5,8 +5,9 @@ const qro = new City('Querétaro');
 console.log(qro.name);
 
 qro.printStats()
-  .then(() => {
+  .then((stats) => {
     console.log('callback after printStats()!');
+    console.log(stats);
   })
 ;
 
