diff --git a/city.js b/city.js
index 45983b6..226b19f 100644
--- a/city.js
+++ b/city.js
@@ -10,10 +10,10 @@ module.exports = class {
   printNameLater () {
     return new Promise((resolve, reject) => {
       setTimeout(() => {
-        reject();
-        //console.log(this.name);
+        //reject();
+        console.log(this.name);
 
-        //resolve(Math.random());
+        resolve(Math.random());
       }, 1000);
     });
   }
