diff --git a/city.js b/city.js
index e8fb7f6..677678b 100644
--- a/city.js
+++ b/city.js
@@ -10,9 +10,10 @@ module.exports = class {
     printNameLater() {
         return new Promise((resolve, reject) => {
             setTimeout(() => {
-                console.log(this.name);
+                reject();
+                //console.log(this.name);
 
-                resolve(Math.random());
+                //resolve(Math.random());
             }, 1000);
         });
     }
