diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index e1661c6..118ee43 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -64,7 +64,7 @@
 
         <ul class="list-unstyled">
             {% for comment in question.comments %}
-                <li class="mb-4 js-comment-container" data-vote-url="{{ path('app_comment_vote', {id: comment.id}) }}">
+                <li class="mb-4 js-comment-container" data-controller="comment" data-vote-url="{{ path('app_comment_vote', {id: comment.id}) }}">
                     <div class="d-flex justify-content-center">
                         <div class="mr-2 pt-2">
                             <img src="{{ asset('images/tisha.png') }}" width="50" height="50" alt="Tisha avatar">
