diff --git a/assets/js/app.js b/assets/js/app.js
index a64897e..d8f7eea 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -18,8 +18,10 @@ if (comments.length > 0) {
   });
   */
 
+  /*
   const submitAnswerButton = document.querySelector('.js-submit-answer');
   submitAnswerButton.addEventListener('click', () => {
     comments[0].parentElement.prepend(comments[0].cloneNode(true))
   });
+   */
 }
