diff --git a/assets/js/app.js b/assets/js/app.js
index 9c7c0ee..a64897e 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -10,11 +10,13 @@ application.load(definitionsFromContext(context));
 
 const comments = document.querySelectorAll('.js-comment-container');
 if (comments.length > 0) {
+  /*
   import('./components/comment').then(Comment => {
     comments.forEach(element => {
       new Comment.default(element);
     });
   });
+  */
 
   const submitAnswerButton = document.querySelector('.js-submit-answer');
   submitAnswerButton.addEventListener('click', () => {
