diff --git a/assets/js/controllers/comment_controller.js b/assets/js/controllers/comment_controller.js
index b9026ca..b21c63f 100644
--- a/assets/js/controllers/comment_controller.js
+++ b/assets/js/controllers/comment_controller.js
@@ -1,8 +1,4 @@
 import { Controller } from 'stimulus';
-import Comment from '../components/comment';
 
 export default class extends Controller {
-  initialize () {
-    new Comment(this.element);
-  }
 }
